.github/workflows/dymos_docs_workflow.yml

```diff
@@ -102,7 +102,11 @@ jobs:
           echo "Install jax"
           echo "============================================================="
           if [[ "${{ matrix.JAX }}" == "latest" ]]; then
-            python -m pip install jaxlib jax
+            {
+              python -m pip install jaxlib
+            } || {
+              echo "Failed to install jaxlib..."
+            }
```

Comment:
oops.. looks like I left out a line here:
```
            python -m pip install jax
```
(as seen in dymos_tests_workflow.yml)

I'll add this to my branch or you can add it here.. sorry

Reply:
my branch updated if you want to merge it in